packages/website/src/features/Packages/FunctionInput.tsx

```diff
@@ -119,14 +119,14 @@ export const FunctionInput: FC<{
                     <CloseIcon name="close" color="red.500" />{' '}
                   </Box>
                 )}
+
+                <Link onClick={add} float="right" padding="0.5rem">
+                  <AddIcon name="add" color="green.500" />
+                </Link>
```

Comment:
The fix for the '+' button floating is making it showed repeated:
<img width="419" alt="Screenshot 2024-07-09 at 13 15 36" src="https://github.com/usecannon/cannon/assets/558901/9996fd51-1758-459a-b488-f3e3e2229317">